src/System.Management.Automation/engine/CmdletParameterBinderController.cs

```diff
@@ -3484,7 +3484,7 @@ private bool BindPipelineParametersPrivate(PSObject inputToOperateOn)
                         validParameterSets);
             }
 
-            s_tracer.WriteLine("aParameterWasBound = {0}", aParameterWasBound);
+            s_tracer.WriteLine("aParameterWasBound = {0}", aParameterWasBound.ToString());
```

Comment:
It's worth reviewing tracing messages like this to see if there are still useful. Without any context, this one doesn't look useful.

Reply:
Will do the review, at least in the parameter binding code :)

Reply:
I suggest to add new overload like `internal void WriteLine(string format, bool arg1)`
To save time I created #10052. There is 18 references to the new overload and up to 46 references for other value types.
So we could revert the change in current PR.

Reply:
OK, I will revert this line.